dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -76,6 +115,13 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 	 */
 	const renderAds = !front.isAdFreeUser;
 
+	if (front.isNetworkFront) {
+		getMobileAddPosition(
+			front.isNetworkFront,
+			front.pressedPage.collections,
+		);
+	}
```

Comment:
Right now, `mobileAddPosition` is defined in the root of the file, this is probably not best practice & potentially could cause issues with server side rendering (ad slots may persist between renders depending on how things work under the hood). 

Perhaps a refactor so that `getMobileAddPosition` returns an array would work better?

```js
// This can be declared at the root of the file
const getMobileAdPositions = (
	isNetworkFront: boolean | undefined,
	collections: DCRCollectionType[],
) => {
    let positions = number[]
	const merchHighPosition = getMerchHighPosition(
		collections.length,
		isNetworkFront,
	);

	collections.forEach((el, ind, arr) => {
		if (
			(ind === 0 && el.collectionType === 'fixed/thrasher') ||
			ind === merchHighPosition ||
			ind + 1 === merchHighPosition ||
			arr[ind + 1]?.collectionType === 'fixed/thrasher'
		) {
			return false;
		} else if (ind % 2 === 0) {
			positions.push(ind);
		}
		return false;
	});
    // separate suggestion for this incoming
	if (positions.length > 10) {
		positions.length = 10;
	}
};

// ....

export const FrontLayout = ({ front, NAV }: Props) => {
     // ...
     const mobileAdPositions = front.isNetworkFront ? getMobileAdPositions(...) : []
    // ...
```